api/axis.py

```diff
@@ -1725,6 +1725,8 @@ def do_fetch(self):
         self._content = str(manifest)
         self._content_type = manifest.MEDIA_TYPE
         self._content_expires = expires
+        # Never used in this object, required for consistency
+        self.content_link_redirect = False
```

Comment:
It seems like this should be handled in `APIAwareFulfillmentInfo.__init__`, so this feature has consistent behavior for new integrations. I noticed that `APIAwareFulfillmentInfo.__init__` doesn't make a call to `super().__init__`, which seems problematic and might be why this property was not present as expected.

Reply:
This is a fair observation, although Axis360FulfillmentInfo is the only one using this right now, this will make it more future proof